dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -655,7 +634,39 @@ private JSONObject convertJSONObjectToArray(String keyName, JSONObject entryJson
                 entryJson.put(keyName, reqArray);
             }
         }
-        return entryJson;
+    }
+
+    private void convertRequirementsAndHintsToArray(JSONObject entryJson) {
+
+        convertJSONObjectToArray("hints", entryJson);
+        convertJSONObjectToArray("requirements", entryJson);
+
+        // for each step, convert the hints and requirements in each step and in the entry that the step runs.
+        if (entryJson.has("steps")) {
```

Comment:
Could create a constant/var for "steps"

Reply:
So, I understand the rationale for creating constants for duplicate strings that might change, like error messages.  But this string is part of a specification, it will never change (caveat: never say never).  What does creating a constant/var do, besides obfuscating the actual value?

Reply:
I suppose one could argue that you could attach some meaning to the value by creating a constant/var and embedding the meaning in the constant/var's name.  However, I'm not sure that's necessary in this code, it's pretty easy to infer the meaning of "steps", especially given the comment above.

Reply:
Comment applies to a lot of hard-coded keys that are strewn throughout the file, e.g., "run".  I know, it's hard to misspell "run", but still.